api/src/main/java/org/openmrs/module/reporting/report/ReportRequestPageDTO.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.reporting.report;
+
+import java.util.List;
+
+public class ReportRequestPageDTO {
```

Comment:
I'd prefer to see us not introduce this new ReportRequestPageDTO class, but to just have separate DAO methods - one to get a count of all requests that match the criteria, and one to get the current list of requests that match the criteria.  Creating a separate shared method to create the common criteria that would be used would be fine.

Reply:
Removed.